### ISSUE TITLE
hitting back button from search resets timeline position to top

### ISSUE THREAD

**Issue by deutrino** [stom79/mastalab#99]
Steps to reproduce:

1. Scroll down in home timeline 
2. Open sidebar 
3. Run a search 
4. Hit back button to return to timeline 

What actually happens:

Position in timeline is reset to the top

What I expect to happen:

Position in timeline should not have changed. 

Mastalab 1.6.4.1 via #77

**Comment by stom79** [comment]
I didn't manage to reproduce this bug. Did this bug happen in every timelines? Did you receive new toots in the timeline when this bug occurred?

**Comment by deutrino** [comment]
I've reproduced it on 1.6.5.1 but it is not 100% reproducible. I'll try to better characterize it.

What I did:

1. opened home timeline
2. scrolled a ways
3. opened sidebar
4. tapped 'search'
5. tapped the text entry box in the search screen (this should be autofocused, btw)
6. entered text and ran the search
7. hit the back button from the results without tapping any of them

In one case, my position had been reset to the top. In another case, it hadn't moved.
